src/styles/clock.scss

```
@@ -0,0 +1,83 @@
+.clock {
+  position: relative;
+  width: 600px;
+  height: 600px;
+
+  margin: 150px auto;
+
+  box-sizing: border-box;
+  border-radius: 50%;
+  border: 9px solid #424242;
+
+  &::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+
+    border-radius: 50%;
+    border: 4px solid #ff4040;
+  }
+
+  &__center {
+    position: absolute;
+    top: calc(50% - 13px);
+    left: calc(50% - 13px);
+    height: 36px;
+    width: 36px;
+
+    background-color: #424242;
+    border-radius: 50%;
+  }
+
+  &__sec {
+    position: absolute;
+    top: 50px;
+    left: calc(50% + 4px);
+
+    height: 250px;
+    width: 1px;
+
+    background-color: #424242;
+    transform-origin: bottom;
+    animation: rotate 60s infinite linear;
```

create variable for 1 minute value

---

also, linear is default value, remove it everywhere
```suggestion
    animation: rotate 60s infinite;
```